go/bench.go

```diff
@@ -56,8 +56,14 @@ func (b *bench) spawn() {
 
 func (b *bench) result() map[string]*result {
 	resMap := map[string]*result{}
+	var d int
```

Comment:
add a fixme comment saying this needs to be fixed for range and is used for throughput calc